src/Disks/ObjectStorages/IObjectStorage.h

```diff
@@ -139,6 +140,7 @@ class IObjectStorage
     virtual void copyObjectToAnotherObjectStorage( /// NOLINT
         const StoredObject & object_from,
         const StoredObject & object_to,
+        const ReadSettings & read_settings,
```

Comment:
Why isn't `write_settings` here?

Reply:
Missed, fixed. (though it is not that important since the only user is the metadata restore for the future that is considered deprecated, but still, the interface should be generic - totally agree!)

Reply:
Thank you!

Reply:
And what about `copyObject`?

Reply:
Oh, missed that as well. Thanks, fixed.